lib/delta_filters.rb

```diff
@@ -0,0 +1,23 @@
+module Inbox
+  class DeltaFilters
+
+    attr_reader :filter_map
+
+    def self.build_exclude_types(object_types)
+      new.build_exclude_types(object_types)
+    end
+
+    def initialize(filter_map: API::OBJECTS_TABLE)
```

Comment:
Any reason you prefer a default value vs. a keyword argument with a default value?